File: CS/WindowsApplication1/Custom Ribbon Control/MyRibbonControl.cs
using DevExpress.XtraBars.Ribbon;
using DevExpress.XtraBars.Ribbon.ViewInfo;

namespace WindowsApplication1 {
    [System.ComponentModel.DesignerCategory("")]
    public class MyRibbonControl : RibbonControl {
        protected override RibbonViewInfo CreateViewInfo() {
            return new MyRibbonViewInfo(this);
        }
    }
}

File: VB/WindowsApplication1/Custom Ribbon Control/MyRibbonControl.vb
Imports DevExpress.XtraBars.Ribbon
Imports DevExpress.XtraBars.Ribbon.ViewInfo

Namespace WindowsApplication1
    <System.ComponentModel.DesignerCategory("")> _
    Public Class MyRibbonControl
        Inherits RibbonControl

        Protected Overrides Function CreateViewInfo() As RibbonViewInfo
            Return New MyRibbonViewInfo(Me)
        End Function
    End Class
End Namespace
